apps/condo/domains/property/components/panels/Builder/BuildingPanelEdit.tsx

```diff
@@ -379,6 +383,8 @@ export const BuildingPanelEdit: React.FC<IBuildingPanelEditProps> = (props) => {
                             addUnit: <UnitForm Builder={Map} refresh={refresh}/>,
                             editSection: <EditSectionForm Builder={Map} refresh={refresh}/>,
                             editUnit: <UnitForm Builder={Map} refresh={refresh}/>,
+                            removeBasement: <RemoveBasementForm Builder={Map} refresh={refresh} />,
```

Comment:
all of props are elements of an object actually, so `Builder` should be `builder` actually